### PR TITLE
JUSI-1617

### PR DESCRIPTION
**Pull request by cganley1**
removed astropy variable causing issue with data grab, added boto3 for data grab. I have a clean build locally.

### CODE REVIEW

**Review by msanchezst — COMMENTED** [review]
Did you test the MAST query in your local build?

File: deployments/roman/default-home-contents/.astropy/config/astroquery.cfg
Comment (msanchezst) [inline deployments/roman/default-home-contents/.astropy/config/astroquery.cfg:1]: Interesting, I didn't know we were pushing this setting

**Review by msanchezst — APPROVED** [review]
Awesome, thanks!

**Comment by cganley1** [comment]
Reverted change, git does not allow empty directories.

**Review by msanchezst — APPROVED** [review]
Looks good to me. Thanks

**Pull request merged**